target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8065-rt4230w-rev6.dts

```diff
@@ -0,0 +1,303 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "qcom-ipq8065.dtsi"
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Askey RT4230W REV6";
+	compatible = "askey,rt4230w-rev6", "qcom,ipq8065", "qcom,ipq8064";
+
+	memory@0 {
+		reg = <0x42000000 0x3e000000>;
+		device_type = "memory";
+	};
+
+	aliases {
+		led-boot = &ledctrl3;
+		led-failsafe = &ledctrl1;
+		led-running = &ledctrl2;
+		led-upgrade = &ledctrl3;
+	};
+
+	chosen {
+		bootargs = "rootfstype=squashfs noinitrd";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&button_pins>;
+		pinctrl-names = "default";
+
+		reset {
+			label = "reset";
+			gpios = <&qcom_pinmux 54 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+			debounce-interval = <60>;
+			wakeup-source;
```

Comment:
AFAICT, this is not supported in the downstream driver. Given we won't be using the upstream driver, drop this please

Reply:
So I just completely delete these? What about the reset button? I didn't add any extra kmods or anything and they just work already so something's making it work.

Reply:
You can just drop the `wakeup-source` line. The *downstream driver* refers to `kmod-gpio-button-hotplug`, which is added to the whole target by default. You can also drop the `debounce-interval` line unless you have a good reason to keep it. That's what I meant in https://github.com/openwrt/openwrt/pull/3115#discussion_r491644968.

Reply:
I removed the two lines @mans0n mentioned